### PR TITLE
python-pyserial: Update to version 3.4

### PR DESCRIPTION
**Pull request by kvuorine**
Maintainer: Micke Prag <micke.prag@telldus.se>
Compile tested: bcm2708 OpenWRT GIT 0f063f8
Run tested: bcm2708 OpenWRT GIT 0f063f8

Description:
Update to version 3.4
Add build for python3
Add source packages

Signed-off-by: Kimmo Vuorinen <kimmo.vuorinen@gmail.com>

### CODE REVIEW

**Comment by hnyman** [comment]
cc maintainer  @mickeprag

**Comment by hnyman** [comment]
@commodo 
any generic python view to this?

File: lang/python/python-pyserial/Makefile [hunk 1]
```diff
@@ -8,43 +8,62 @@
...
 include ../python-package.mk
+include ../python3-package.mk
 
-define Package/python-pyserial
-  SUBMENU:=Python
-  SECTION:=lang
+PKG_UNPACK:=$(HOST_TAR) -C $(PKG_BUILD_DIR) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
+
+define Package/python-pyserial/Default
+  SECTION:=lang-python
```
Comment (commodo) [inline lang/python/python-pyserial/Makefile]: i'm curious about this; most Python packages have `SECTION:=lang` ; there are a few other packages that seem to define `SECTION:=language-python` and `SECTION:=lang-python`; I don't have a strong preference here [about which one to use]; but I would vote for consistency [whether it's `SECTION:=lang`, `SECTION:=lang-python` or `SECTION:=language-python` ]

**Comment by commodo** [comment]
otherwise looks good;
the conversion looks spot-on to the way Python + Python3 packages should be defined

thanks :)

**Comment by hnyman** [comment]
I will merge this as commodo likes the python style. 

Regarding the SECTION, I am not even sure if that has actually impact anywhere. Or if that is even visible anywhere.

**Pull request merged**